src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/Dataflow/FlowAnnotations.cs

```diff
@@ -16,6 +16,8 @@
 
 using Debug = System.Diagnostics.Debug;
 using WellKnownType = Internal.TypeSystem.WellKnownType;
+using System.Reflection;
```

Comment:
Nit: could you organize the usings the way they're organized (also in other files)?